source/modules/configuration/asset_definitions/asset_definitions.rst

```diff
@@ -0,0 +1,407 @@
+   Examples : API calls, webhooks, etc.
+   It can be personnalized, but some words are reserved such as classes from GLPI like Computer, Monitor, etc.
+   Items linked to the system name "Example" will have the class "GLPI\CustomAsset\ExampleAsset"
+
+After the creation, an error message appears : There is currently no profile with access to items with current definition
+You need to go to :doc:`profiles <#profiles>`
+
+
+Capacities
+----------
+
+Capacities lets you add items such as software, network ports, contacts, etc. to the asset.
+Each asset can be selected and customised as required.
+
+
+.. image:: images/capacities.png
+   :alt: Add new capacities
+   :scale: 37 %
+
+List of the elements that can be linked:
+
+* Antivirus
+* Appliances
+* Certificates
+* Components
+* Connections
+* Contracts
+* Databases instances
+* Documents
+* Domains
+* Financial and administrative information
+* Global search
+* Historical
+* Impact analysis
+* Inventory
+* Knowledge base
+* Manual links / External links
+* Network ports
+* Notes
+* Operating systems
+* Plugs
+* Projects
+* Racks
+* Remote management
+* Reservations
+* Sockets
+* Software
+* Virtualization
+* Volumes
+
+
+Fields
+------
+
+The fields tab is used to add additional fields.
+You can customise them by indicating whether they should be text, URL, date, etc.
+
+Create a custom field
+~~~~~~~~~~~~~~~~~~~~~
+
+* To add a new field, click on **+ New field**
+* Fill in the required fields
+
+.. image:: images/fields.png
+   :alt: Add a custom field
+   :scale: 37 %
+
+* **Label**: name which will be displayed on the equipment sheet
+* **System name**: The system name field corresponds to what will be used when development is involved. Examples: API calls, webhooks, etc.
+  In the legacy API, the field name is prefixed by ``custom_``; to avoid conflicts with standard fields.
+* **Type**: string, date, URL, dropdown, yes/no, text, date and time, number. Cannot be modified once saved
+* **Full width**: authorises the field to be extended along the entire length of the form
+* **Mandatory**: make it mandatory or not to fill in the field before saving the equipment file
+* **Readonly for these profiles**: select one or more profiles with read-only access to this field.
+  The permissions in the profiles tab take precedence over this field.
+* **Hidden for these profiles**: select one or more profiles whose fields will be hidden
+  The authorisations in the profiles tab take precedence over this field. It will therefore be visible to a profile even if it is selected in this field.
+* **Default values**: specify a default value
+
+.. note:: For dropdown list, you need to select the item in the list. You can specify if multiple selection is authorised. You can select a default value if necessary
+
+   .. image:: images/dropdown_field.png
+      :alt: Dropdown list selection
+      :scale: 61%
+
+   .. collapse:: Dropdown list details

```

Comment:
I assume this was directed to be exhaustive, but I am making another comment here anyways that this will be annoying to maintain and makes it annoying to read the documentation when such a long list is always visible.

Reply:
It also doesn't mention the fast you can create custom dropdown types and those will be available as selections here.